src/module_ncio.f90

```diff
@@ -133,32 +157,27 @@ subroutine nccheck(status,halt,fname)
        if (stopit) stop 99
     end if
   end subroutine nccheck
-
-  !>
+  !> get Dimension object given name
```

Comment:
Always capitalize and use punctuation for the first (and every other) sentence in the docs.